frontend/ui/widget/doublespinwidget.lua

```diff
@@ -23,29 +23,30 @@ local WidgetContainer = require("ui/widget/container/widgetcontainer")
 local _ = require("gettext")
 local Screen = Device.screen
 
-local HyphenationLimitsWidget = InputContainer:new{
-    title_text = _("Hyphenation limits"),
+local DoubleSpinWidget = InputContainer:new{
+    title_text = "",
     title_face = Font:getFace("x_smalltfont"),
+    info_text = "",
     width = nil,
     height = nil,
-    -- Min (2) and max (10) values are enforced by crengine
     left_min = 1,
     left_max = 10,
     left_value = 2,
     left_default = nil,
+    left_text = _("Left"),
     right_min = 1,
     right_max = 10,
     right_value = 2,
     right_default = nil,
+    right_text = _("Right"),
+    button_default_text = _("Use language defaults"),
```

Comment:
This text belongs to readerhyphenation, it's really not a good default for DoubleSpinWidget :)
Either a more generic `_("Use defaults")`, or make that additional button optional if that value is nil (and use nil as the default).